example/java/prepare.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+cd $(dirname $0)
+
+# generate iroha lib
+if [[ $1 -eq 0 && ${IROHA_HOME} -eq 0 ]] ; then
+    echo 'Neither IROHA_HOME nor path to CMakeLists.txt is set'
+    exit 0
+fi
+cmake_path=${1,-${IROHA_HOME}}
```

Comment:
what a weird construction?
At what cases it is required to set `cmake_path` to 0?